### PR TITLE
Fix erroneous use of parameters in callbacks

### PR DESCRIPTION
**Pull request by KamuelaFranco**

### CODE REVIEW

File: test/onboarding/onboarding.test.js [hunk 1]
```diff
@@ -741,7 +718,7 @@ describe('Onboarding', () => {
             const headers = new Headers(headersMap);
             const body = JSON.stringify({
                 type: 'FSPIOP_CALLBACK_URL_TRANSFER_ERROR',
-                value: 'http://dfsp/transfers/dee18631-6c40-438a-bdd0-67bac5e4e1c2/error',
+                value: 'http://dfsp/transfers/{{transferId}}/error',
```
Comment (msk-) [inline test/onboarding/onboarding.test.js:721]: Is this correct? Specifically `http://dfsp`?

Comment (msk-) [reply test/onboarding/onboarding.test.js:721]: I guess this question applies to the other places where this is present also.

Comment (KamuelaFranco) [reply test/onboarding/onboarding.test.js:721]: @msk- as represented by the given `const dfspCallbackUrl = 'http://dfsp';`, yes.

File: src/onboarding/onboarding.js [hunk 2]
```diff
@@ -255,7 +251,7 @@ function addCallbackParticipantPut({
     headers.append('FSPIOP-Source', fspiopSource);
     headers.append('Authorization', `Bearer ${authToken}`);
 
-    const fullCallbackUrl = `${dfspCallbackUrl}/participants/${dfspPartyIdType}/${dfspPartyId}`;
+    const fullCallbackUrl = `${dfspCallbackUrl}/participants/{{partyIdType}}/{{partyIdentifier}}`;
```
Comment (Kalaston) [inline src/onboarding/onboarding.js:254]: you removed the params for `dfspPartyId` and `dfspPartyIdType`, but didn't add `partyIdType` and `partyIdentifier`. I think these will be `undefined`.

Comment (KamuelaFranco) [reply src/onboarding/onboarding.js:254]: No @Kalaston, these are interpreted by the server as templates. The server fills them in when it creates the callback URL's. That's what has been explained to me by @shashi165.

File: test/onboarding/onboarding.test.js (hunk 1 above)
Comment (msk-) [reply test/onboarding/onboarding.test.js:721]: Agh, man, I didn't realise that was in tests. That's embarrassing.

**Pull request merged**